org.mwc.debrief.core/src/org/mwc/debrief/core/editors/PlotOutlinePage.java

```diff
@@ -797,7 +797,7 @@ private void contributeToActionBars()
   public void createControl(final Composite parent)
   {
 
-    _treeViewer = new MyTreeViewer(parent, SWT.MULTI | SWT.H_SCROLL
+    _treeViewer = new MyTreeViewer(parent, SWT.SINGLE | SWT.H_SCROLL
```

Comment:
@ayeshare - I think we need to be able to do multi-select, partly to request an XY graph of multiple tracks.  Is it essential to have SWT.SINGLE for this fix to work?

Reply:
Aah, I've switched it back to MULTI, and it works ok.  

If there is a problem, we'll have to come back & fix it.